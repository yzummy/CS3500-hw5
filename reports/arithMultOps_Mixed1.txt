
___Entering new scope...				      <
							      <
TOKEN: FLOATCONST 		 LEXEME: 3.14		      <
CONST -> FLOATCONST					      <
FACTOR -> CONST						      <
TOKEN: DIV 		 LEXEME: /			      <
MULT_OP -> /						      <
TOKEN: INTCONST 		 LEXEME: 10		      <
CONST -> INTCONST					      <
FACTOR -> CONST						      <
MULT_OP_LIST -> epsilon					      <
MULT_OP_LIST -> MULT_OP FACTOR MULT_OP_LIST		      <
TERM -> FACTOR MULT_OP_LIST				      <
ADD_OP_LIST -> epsilon					      <
SIMPLE_ARITHLOGIC -> TERM ADD_OP_LIST			      <
ARITHLOGIC_EXPR -> SIMPLE_ARITHLOGIC			      <
EXPR -> ARITHLOGIC_EXPR					      <
START -> EXPR						      <
							      <
---- Completed parsing ----					---- Completed parsing ----

 Value of the expression is: 3.14			      |	Value of the expression is:   0.31
							      <
___Exiting scope...					      <
							      <
