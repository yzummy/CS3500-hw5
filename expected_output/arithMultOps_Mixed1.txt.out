
---- Completed parsing ----

Value of the expression is:   0.31
